src/javascript/app_2/App/Components/Elements/PositionsDrawer/result-details.jsx

```diff
@@ -19,52 +23,61 @@ class ResultDetails extends React.PureComponent {
 
     render() {
         const {
-            barrier,
             contract_end_time,
-            contract_start_time,
+            contract_info,
             duration,
             duration_unit,
-            entry_spot,
+            exit_spot,
             has_result,
-            id_sell,
-            tick_count,
         } = this.props;
         if (!has_result) return null;
         return (
             <React.Fragment>
-                <div className='result-details__separator' />
                 <div className={classNames('result-details__wrapper', {
                     'result-details__wrapper--is-open': this.state.is_open,
                 })}
                 >
                     <div className='result-details__grid'>
                         <ResultDetailsItem
-                            label={localize('Reference ID')}
-                            value={id_sell}
+                            label={localize('Ref. ID (Buy)')}
+                            value={contract_info.transaction_ids.buy}
                         />
                         <ResultDetailsItem
-                            label={localize('Duration')}
-                            value={tick_count ? `${tick_count} ${localize('ticks')}` : `${duration} ${duration_unit}`}
+                            label={localize('Ref. ID (Sell)')}
+                            value={contract_info.transaction_ids.sell}
                         />
                     </div>
                     <div className='result-details__grid'>
                         <ResultDetailsItem
-                            label={localize('Barrier')}
-                            value={barrier ? barrier.toString() : ' - '}
+                            label={localize('Duration')}
+                            value={(contract_info.tick_count > 0) ?
+                                `${contract_info.tick_count} ${(contract_info.tick_count < 2) ? localize('tick') : localize('ticks')}`
```

Comment:
should we check specifically `=== 1`? since if we ever have 0, it should also be 0 ticks